lib/viewing/Layout.js

```diff
@@ -28,6 +22,8 @@ var Layout = View.extend({
 
     updateMetatagsOnClientRender: false,
 
+    fetchData: noop,
```

Comment:
the noop in brisket came with self documenting functions like "noop.thatWillBeCalledWith". we don't really need that